index.js

```diff
@@ -197,3 +197,19 @@ function myFunction() {
 }
 
 window.onscroll = function () { myFunction(); };
+
+// form validation
+
+const form = document.querySelector('#form');
+const email = document.querySelector('#email');
+const emailInput = email.value;
+const msg = document.querySelector('#errorMsg');
+const regex = /[a-z]/;
+
+form.addEventListener('submit', (event) => {
+  if (!regex.test(emailInput)) {
+    event.preventDefault();
+    msg.classList.add('showError');
+    msg.textContent = '*The content of the email field has to be in lower case.';
+  }
+});
```

Comment:
- [ ] Great job so far, but unfortunately, while this function seems to prevent me from sending the form with an uppercase letter in the email, it also seems to not allow me to send even when the email is in lowercase, kindly fix that